schema/Makefile

```diff
@@ -0,0 +1,63 @@
+.DEFAULT_GOAL := build
+
+# include $CORE_VERSION, $API_VERSION, $INGESTION_CONFIG_VERSION
+include .version
+
```

Comment:
versions are pulled from `schema/.version` file

Reply:
Why do we want to do this *and* have a symlink? I figure one or the other should be enough?

Reply:
@jgadling I guess if we have only the symlink, then we won't have anywhere where we explicitly store the version (I don't know if this a real concern we should be worried about). If we only have the .version, the referencing the latest version might be a bit more verbose (where we have to read & parse / load in the .version folder, as opposed to just referencing the latest folder). Thoughts? Thank you!

Reply:
Ahhh, I see. I agree that just setting the version as the real directory name might not be *super* explicit, but it seems good enough. If you want to keep the version file though, can you document the 2-step process for upgrading versions in the schema README?

Reply:
I can go ahead and update everything to use "latest" (so no version file), and then I'll document how to upgrade versions in the README (one-step process, just update the symlink). I'll also add a note how the latest version is just the "latest" folder, which if you really want to know which version that is, you can just `readlink` the symlink, which should anyways just be the highest-versioned folder. Does that sound okay?

Reply:
Now thinking about it, I think we should keep the .version number because ingestion_config/ is the only folder that needs a latest/ folder right now, so might just better to explicitly state it for everything and keep the latest/ folder for ingestion_config/, changing in the future if necessary. The extra step shouldn't be too much of a hassle and I've gone ahead and updated the README with instructions accordingly. Thanks!